Describe Date Range Parameters (#31)
serge-andreyev/dotnet-eud: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/query-parameters.md

---
title: Query Parameters
author: Anna Gubareva
---
# Query Parameters

This document provides information on query parameters and describes how to use parameterized SQL queries to filter data at data source level.

## <a name="overview"></a>Query Parameters Overview

> A query parameter holds an external value that is inserted into an SQL statement before query execution. This value can be either static or dynamically generated by an associated expression.

Reviewer: ```suggestion
A query parameter holds an external value that is inserted into an SQL statement before query execution. This value can be static or an associated expression can generate it dynamically.
```

---
title: Query Parameters
author: Anna Gubareva
---
# Query Parameters

> This document provides general information on query parameters and describes common ways of utilizing parametrized SQL queries to filter data at the data source level.

Reviewer: ```suggestion
This document provides information on query parameters and describes how to use parameterized SQL queries to filter data at data source level.
```
